### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mirrorkeydev**
Rewrote and restructured most of the README!

See it in all of its formatted glory [here](https://github.com/googleinterns/noms/blob/Readme/README.md).

New features include:

- Build badges (however, our only CI action is the linter, so it's a linter badge)
- Table of contents
- Rewrote the motivation section to be clearer and have citations (updated design doc as well)
- Mentioned all technologies used (if I missed any, please let me know!)
- Demo GIFs/images of our app
   - The Gmail screenshots have been scrubbed of most of the interface, to prevent leaking new features and to look cleaner
- Expanded the features we each worked on
- Explicit mention of license + disclaimer

Things I could still add, depending on y'allses opinions:
- Screenshots of about/contact page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,95 @@
-# noms
+# 🍱 noms
+[![Super Linter](https://github.com/googleinterns/noms/workflows/Super%20Linter/badge.svg)](https://github.com/googleinterns/noms/actions)
```
Comment (areetaw) [inline README.md:2]: Amazing!! :)

File: README.md [hunk 2]
```diff
@@ -41,16 +99,29 @@ To run a local server, execute this command:
 mvn package appengine:run
 ```
 
-## Contributors
+## Authors
+Google STEP (Student Training in Engineering Program) Pod #186, Summer 2020
```
Comment (areetaw) [inline README.md:103]: What an awesome pod!

File: README.md [hunk 3]
```diff
@@ -1,37 +1,95 @@
...
+  - [Sign up for email notifications](#sign-up-for-email-notifications)
+  - [Receive email notifications](#receive-email-notifications)
+- [Usage](#Usage)
+- [Authors](#Authors)
+- [Contributing](#Contributing)
+- [License](#License)
+- [Disclaimer](#Disclaimer)
 
-Combine this issue with the reality that at some colleges, up to half of all students regularly experience food insecurity, and getting leftover food into the right hands (and stomachs) becomes not just an environmental necessity, but a humanitarian one as well. However, in attempting to address these issues, college organizations may find they lack the infrastructure to quickly notify and distribute their leftover food to the students who need it most, despite the fact that the students are otherwise ideally located to receive the food.
+## Description
```
Comment (areetaw) [inline README.md:23]: For internal users/reviewers, should we create a separate section with go/links that would be helpful like the app, feedback form, bug form, and our design docs?

Comment (mirrorkeydev) [reply README.md:23]: I'm not sure if go/links are allowed to be public? Like I understand they're not accessible to anyone outside of the corp network but I'm not sure if they're allowed to be mentioned, might have to ask Sanya or Natalie

But if they are, I'd be down to add a section!

Comment (areetaw) [reply README.md:23]: If not go/links, the links itself would be nice!

Comment (mirrorkeydev) [reply README.md:23]: Actually, upon thinking about it more, I think that the design doc itself should be the main hub for these links for a couple reasons:

- Anyone who can see our design doc is already corp, so they'll be able to access all of the links (I imagine random people coming across this repo might be miffed if a whole section of links doesn't work for them)
- Related to the above point, I don't think that the links in the readme would age well; after the internship is over, some of those links won't be relevant anymore (such as the feedback/bug forms, as we won't be able to access responses), and removing them from the readme would require 3 reviewers or removing branch protections
- We can use go/links without issue in the design doc
- The design doc is the main hub of documentation on our project anyway (it branches out to our feature docs)

Lemme know if any of this points seem unreasonable or iffy!

File: README.md [hunk 4]
```diff
@@ -1,37 +1,95 @@
...
+On a college's page, the map and feed allow users to browse food available at their college. The user is present on the map as a blue marker. Filters allow the user to narrow down the type of post.
+
+![](https://user-images.githubusercontent.com/35010111/91066586-bd562680-e5e6-11ea-9248-8b3ef66ad6dc.gif)
+![](https://user-images.githubusercontent.com/35010111/91067113-5e44e180-e5e7-11ea-9026-bf5ca84d6b9c.gif)
+
+
+### Make a new post
+The new post modal features validation for all of its fields and uses the Geocoding Service to automatically translate addresses entered by the user into map coordinates.
+
+![](https://user-images.githubusercontent.com/35010111/91067907-5afe2580-e5e8-11ea-98c9-07d6fd0ae721.gif)
```
Comment (ubahl) [inline README.md:81]: Nice! I like the videos

**Pull request merged**